graphql/version_resolver.go

```diff
@@ -70,7 +70,7 @@ func (r *versionResolver) BuildVariants(ctx context.Context, obj *restModel.APIV
 		obj.Activated = version.Activated
 	}
 
-	if !utility.FromBoolPtr(obj.Activated) {
+	if evergreen.IsPatchRequester(utility.FromStringPtr(obj.Requester)) && !utility.FromBoolPtr(obj.Activated) {
```

Comment:
If we're using this in a lot of places, it seems like it could be easier to make a function on APIVersion called IsPatchRequester that just returns this

Reply:
Done!